test/unit/saveArticles.test.ts

```diff
@@ -81,35 +81,6 @@ describe('saveArticles', () => {
       dump2 = new Dump('', { keepEmptyParagraphs: true } as any, dump.mwMetaData)
     })
 
-    test('Found no empty details elements when they should be stripped in mobile view', async () => {
```

Comment:
The option `keepEmptyParagraphs` has to be tested! Both of these tests should not be removed, but updated... or do I'm missing something?

Reply:
We should modify/adapt.

Reply:
I need your agreement on what to consider as empty paragraphs. Should it be `p` without text content and other empty tags inside? Or just without text content?

Reply:
> I need your agreement on what to consider as empty paragraphs. Should it be `p` without text content and other empty tags inside? Or just without text content?

It should work like before from a user perspective, I can not tell you more.

Reply:
My concern is that it can't work as before because Parsoid doesn't support `details` tags. I can apply changes from https://github.com/openzim/mwoffliner/pull/886/files but trigger parsoid API rather than use mocked HTML.

Reply:
Yes, an other way has to be found to detect and remove empty sections.